GliaWidgets/View/Call/Video/VideoStreamView.swift

```diff
@@ -17,24 +20,47 @@ class VideoStreamView: View {
 
     init(_ kind: Kind) {
         self.kind = kind
-        super.init()
+
+        super.init(frame: .zero)
+
         setup()
     }
 
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
     private func setup() {
         clipsToBounds = true
         layer.cornerRadius = kind == .local ? 6.0 : 0.0
+
+        let panRecognizer = UIPanGestureRecognizer(target: self, action: #selector(pan(_:)))
+        addGestureRecognizer(panRecognizer)
     }
 
     private func replace(
         oldStreamView: StreamView?,
         with streamView: StreamView?
     ) {
         oldStreamView?.removeFromSuperview()
-        guard let streamView = streamView else { return }
-        streamView.scale = .aspectFill
+        guard let streamView = streamView else {
+            show?(false)
+            return
+        }
+        streamView.scale = kind == .remote ? .aspectFit : .aspectFill
         addSubview(streamView)
         streamView.autoPinEdgesToSuperviewEdges()
         streamView.layoutIfNeeded()
+        show?(true)
+    }
+
+    @objc
+    func pan(_ gesture: UIPanGestureRecognizer) {
```

Comment:
perhaps we could make this mehod private since it is not necessary to expose it publically, or do we need it?

Reply:
true, good catch